src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { Link } from './Link'
+
+export default {
+  title: 'Link',
+  parameters: {
+    info: `Link component
+
+    Based on Typography guide: https://designsystem.digital.gov/components/typography/#links`,
+  },
+}
+
+export const Default = (): React.ReactElement => (
+  <p>
+    <Link href={'#'}> This</Link> is a text link.
+  </p>
+)
+
+export const Visited = (): React.ReactElement => (
```

Comment:
Let's just remove the Visited story all together and just comment TODO for now-  since it doesn't work.  My guess is the styling issue is related to custom style themes -  #191

Reply:
I've commented it out